OneCommunityGlobal/HGNRest
# Peterson_Fix_the_500_error_in_editing_last_week

# Description
This PR has been created to fix the bug: ReferenceError: value is not defined.

## priority  Medium

## Related PRS (if any):
This  backend  PR is associated with the following frontend PR: https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2023

## Main changes explained:
The userProfileRouter file has been modified to fix the bug: ReferenceError: value is not defined.

## How to test:
1. check into current branch.
2. do `npm install`, `npm run build`  and `npm start` to run this PR locally.
3. Please adhere to the instructions outlined in the frontend PR.

## Screenshots or videos of changes:
![a](https://github.com/OneCommunityGlobal/HGNRest/assets/73541319/b4fa084e-e229-4325-94c9-16650f14ead9)

## Note:
None
